lib/testing/src/lib/core/utils/local-storage.util.ts

```diff
@@ -37,6 +37,18 @@ export class LocalStorageUtil {
         );
     }
 
+    static async setForm(value: string) {
```

Comment:
it's not related to the Local Storage, so guess "local-storage.util.ts" class is misleading